assets/js/veracode.js

```diff
@@ -104,96 +119,108 @@ jQuery(function () {
 
   const veracodeTable = createTable({
     id: 'veracodeTable',
-    ajaxUrl: `${rootDataUrl}?results=failed`,
+    ajaxUrl: `${rootDataUrl}`,
     orderColumn: 6,
     orderType: 'desc',
     columns,
   })
 
   veracodeTable.on('xhr', function (_e, settings, json) {
     const rawData = Array.isArray(json) ? json : json.data || []
-
     if (!rawData || !rawData.length) return
 
     populateTeamDropdown(rawData)
+    setCheckboxes(currentFilters)
 
-    // Register team filter function
-    $.fn.dataTable.ext.search = []
     $.fn.dataTable.ext.search.push(teamFilterFunction(currentFilters))
     veracodeTable.draw()
   })
 
-  function populateTeamDropdown(data) {
-    const teams = [...new Set(data.map(item => item.team))].sort()
-    const $teamSelect = $('#team')
-    $teamSelect.empty().append('<option value="">All teams</option>')
-
-    teams.forEach(team => {
-      $teamSelect.append(`<option value="${team}">${team}</option>`)
+  // On dropdown update or checkbox change
+  $(
+    '#updateTeam,.environments .govuk-checkboxes__input,.status .govuk-checkboxes__input,.area .govuk-checkboxes__input',
+  ).on('click change', e => {
+    e.preventDefault()
+    //newDataUrl = updateURLParams()
+    currentFilters = getFiltersFromUI()
+    updateURLParams(currentFilters)
+    const newDataUrl = buildAjaxUrl(currentFilters)
+    veracodeTable.ajax.url(newDataUrl).load(() => {
```

Comment:
My only general comment, is a bit more spacing between blocks as it can be difficult to read when there's no empty lines separating blocks.